docs/sql-data-sources-load-save-functions.md

```diff
@@ -41,6 +41,11 @@ name (i.e., `org.apache.spark.sql.parquet`), but for built-in sources you can al
 names (`json`, `parquet`, `jdbc`, `orc`, `libsvm`, `csv`, `text`). DataFrames loaded from any data
 source type can be converted into other types using this syntax.
 
+The available extra options are documented in the API documentation. For basic formats
+(like `json`, `parquet`, `jdbc`, `orc`, `csv`, `text`) you want to check the API documentation of
```

Comment:
I wouldn't make "basic formats" (what's basic formats?) and "other formats" separate here. `libsvm` is a proper built-in source as well. Why don't you like https://github.com/apache/spark/pull/23742#discussion_r254673374 ?